cmd/mayactl/app/command/volume_test.go

```diff
@@ -690,3 +690,69 @@ func TestGetNodeName(t *testing.T) {
 		})
 	}
 }
+
+func TestGetVolumeNamespace(t *testing.T) {
+	tests := map[string]struct {
+		expectedOutput string
+		Volume         VolumeInfo
+	}{
+		"Fetching volume namespace when present in response": {
+			Volume: VolumeInfo{
+				Volume: v1alpha1.CASVolume{
+					ObjectMeta: metav1.ObjectMeta{
+						Namespace: "default",
+					},
+				},
+			},
+			expectedOutput: "default",
+		},
+		"Fetching volume namespace when not present in response": {
+			Volume: VolumeInfo{
```

Comment:
The method receiver is a pointer in source code whereas here this is not. Can we have this as pointer as well?